andela/travel_tool_front
# #161543193 Fix authentication issues

#### What does this PR do?
This PR fixes the `Add user modal` issue which occurs immediately after opening the `edit modal`  on the `travel team page`.

#### Description of Task to be completed?
- Update the `openModal` method for the `add user` button click
- Update tests to suit implementation

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- Install dependencies and start the  app - `make start`
-  [Start](https://github.com/andela/travel_tool_back/pull/139)  the backend.
- Navigate to [travela](http://travela-local.andela.com:3000) in your browser and login into the app. 
- Open your browser's `dev tool`
- Click on the `application tab`
- Under `Storage` select `cookies`
- Modify the value of the `jwt-token`
- Reload the browser
- The browser should redirect to the `login page` with a toast message that says `Invalid token`

#### Any background context you want to provide?
- Ensure you have the `REACT_APP_JWT_PUBLIC_KEY` in the `.env` file

#### What are the relevant pivotal tracker stories?
[#161543193](https://www.pivotaltracker.com/story/show/161543193)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
